charts/cnwan-operator/README.md.gotmpl

```diff
@@ -0,0 +1,143 @@
+# CN-WAN Operator Helm Chart
+
+{{ template "chart.badgesSection" . }}
+
+{{ template "chart.description" . }}
+
+## Useful links
+
+[Project's Homepage]({{ template "chart.homepage" . }})
+[Documentation](https://github.com/CloudNativeSDWAN/cnwan-operator#documentation)
+
+**Sources**:
+{{ template "chart.sourcesList" . }}
+
+## Requirements
+
+**Kubernetes Version**: `{{ template "chart.kubeVersion" . }}`
+
+{{ template "chart.requirementsTable" . }}
+
+The `bitnami/etcd` dependency is only required if you intend to install the
+operator together with etcd. Otherwise, i.e. if you have etcd already installed
+or do not want to use etcd, it won't be needed.
+
+## Values table
+
+The behavior of the operator and the ways it connects to and manages a service
+registry can be fine tuned by setting the parameters of `operator` according
+to your needs -- e.g. by setting a value to `operator.namespacePolicy` -- and
+those of the service registry according to the one you are going to use -- e.g.
+`operator.etcd` or `operator.googleServiceDirectory`.
+
+Values that start with `etcd` -- e.g. `etcd.auth.rbac.enabled` will configure
+the etcd cluster that will be installed along with the operator in case you set
+`operator.etcd.install` to `true`, otherwise they will be ignored.
+
+We highly recommend you to read the
+[official documentation](https://github.com/CloudNativeSDWAN/cnwan-operator/blob/master/docs/configuration.md)
+to better understand the operator's settings.
+
+Finally, look at [examples](#examples) to learn more.
+
+{{ template "chart.valuesTable" . }}
+
+***
+
+## Examples
+
+The following are quickstarts/examples to demonstrate how to deploy the
+operator and configure it to use a certain service registry.
+
```

Comment:
I would also add a quick section here to mention that in addition to using `--set` on the CLI, user can also create a custom `values.yaml` file of their own with their settings, which is more readable, especially if they want to override/set a lot of values.

Reply:
Done, lmk if it's better